### PR TITLE
refactor: refactoring storage status and add quota per storage volume

### PR DESCRIPTION
**Pull request by Sujin-Kim1**
resolves https://github.com/lablup/giftbox/issues/452
## What's changed
- number of folders
  - number of created user folders / keypairs's `max_vfolder_count`
  - number of project folders / Invited folders
- Quota per storage volume
  - host selector 
  - project quota progress bar (`usage_bytes` / `hard_limit_bytes`)
  - user quota progress bar (`usage_bytes` / `hard_limit_bytes`)
 
## Screenshots

Image

Image

For quota supported

Image

### CODE REVIEW

**Comment by yomybaby** [comment]
Let's make a StorageSelector Component. 
- send baiClient.vfolder.list_hosts request (use suspense option)
- inputs(props)
  -  value : host name | volume info type
  -  onChange: `(hostName:string, volumInfo: {}) => void`
  - `autoSelectDefault` prop(optional): auto selecting default host

**Comment by adrysn** [comment]
How about auto-select the first host volume for the quota information? For a single-volume system, it would be much more convenient for users not to select the host every time to see the quota information.

**Comment by Sujin-Kim1** [comment]
@adrysn reflected it!

File: resources/i18n/ko.json [hunk 1]
```diff
@@ -751,7 +752,14 @@
...
     "NoStorageDescriptionFound": "별도 설명이 없습니다.",
-    "DialogModelFolder": "모델을 선택하여 실행하거나, 또는 모델을 관리할 수 있습니다."
+    "DialogModelFolder": "모델을 선택하여 실행하거나, 또는 모델을 관리할 수 있습니다.",
+    "QuotaPerStorageVolume": "스토리지 볼륨별 가용량",
+    "NumberOfFolders": "폴더 개수",
+    "SelectStorageHost": "호스트를 선택하세요",
+    "Used": "사용됨",
+    "Limit": "제한",
+    "HostDetails": "선택한 스토리지 호스트의 가용량 정보를 보여줍니다.",
+    "ProjectFolder": "그룹 폴더"
```
Comment (adrysn) [inline resources/i18n/ko.json]: Let's change "그룹" -> "프로젝트".

**Review by adrysn — APPROVED** [review]
LGTM. Please just change a terminology: "그룹" -> "프로젝트".

**Pull request merged**